sparkbox/apprentice-at
# chore: fixes an issue where local installs fail b/c of node-sass

**The problem:**
After running `npm install`, our postinstall script runs, building the static site. During this process we were running into errors related to an older version of node-sass, which were preventing the site from building.

**Steps to reproduce**
1. Pull down the master branch of this project.
2. Run `npm install`
3. Verify that the install fails.
4. Cry.

**Description of changes**

- Updated `grunt-sass` to a newer version which uses a newer version of `node-sass` (fixing the build errors).
- Checked in `package-lock.json` which should help us have repeatable local installs going forward.
- Updated the environment setup instructions to use `npm ci`, to ensure that local installs are based on the `package-lock.json` file (and not installing newer versions of things by accident).

**Testing steps**
1. Pull down this branch
2. Delete your `node_modules` folder, if you have one.
3. Run `npm ci`
4. Confirm that the install succeeds without errors.
5. Run the site locally and confirm that things look good (`cd dist && heroku local`).
